using-valtimo/document/custom-case-headers.md

```diff
@@ -0,0 +1,86 @@
+# Custom case headers
+
+Instead of de default case header which only shows a case name, each case header can be customized, using a
+configuration in the environment file of a front-end implementation.
+
+The custom header consists of blocks that use the Bootstrap column classes to define the block sizes. In this way, any
+layout can be created.
+
+| <!-- -->                                          |
+|---------------------------------------------------|
+| ![Custom case header](img/custom-case-header.png) |
+| <center>A custom case header</center>             |
+
+## Defining a custom case header
+
+For the above screenshot, the configuration in the environment looks like this:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    customDossierHeader: {
+        aansprakelijkstelling: [
+            {
+                labelTranslationKey: 'aansprakelijkstelling',
+                columnSize: 12,
+                textSize: 'xl',
+                customClass: 'font-weight-bold'
+            },
+            {
+                labelTranslationKey: 'aansprakelijkstelling.naam',
+                propertyPaths: ['aanvrager.persoonsgegevens.voorletters', 'aanvrager.persoonsgegevens.achternaam'],
+                columnSize: 6,
+            },
+            {
+                propertyPaths: ['aanvrager.adres.woonplaats'],
+                columnSize: 3,
+                textSize: 'md',
+                customClass: 'text-danger'
+            },
+            {
+                columnSize: 3
+            },
+            {
+                labelTranslationKey: 'aansprakelijkstelling.adres',
+                propertyPaths: ['aanvrager.adres.adres', 'aanvrager.adres.huisnummer', 'aanvrager.adres.postcode', 'aanvrager.adres.woonplaats'],
+                columnSize: 6,
+                textSize: 'sm',
+            },
+            {
+                labelTranslationKey: 'aansprakelijkstelling.telefoonnummer',
+                propertyPaths: ['aanvrager.persoonsgegevens.telefoonnummer'],
+                columnSize: 3,
+                textSize: 'sm',
+                noValueText: 'Nog niet bekend'
+            },
+        ]
+    }
+    ...
+};
+  ```
+
+`customDossierHeader` is the main object that holds all custom case header configurations. Inside of this object, for
+each case for which a custom case header is to be defined, an array can be added under a key with the case id. In the
```

Comment:
i would change `case` -> `case type` and `an array can be added` -> `an array is added`.

i'm also not sure about the second part of that sentence. You mention an array having a key. I feel the text itself is not very clear, but together with the example users should be able to figure it out.

Optionally change it to something like this?
```
Inside of this object, for each case for which a custom case header is to be defined, an array can be added under a key with the case id.
```
to
```
The `customDossierHeader` contains a property for each case type that requires a custom header. The key is the name of case definition and the value an array of `customDossierHeaderItem` elements.
```